CMakeLists.txt

```diff
@@ -1,11 +1,16 @@
 cmake_minimum_required(VERSION 3.19 FATAL_ERROR)
 
 cmake_policy(SET CMP0091 NEW)
+if(POLICY CMP0135)
+  cmake_policy(SET CMP0135 NEW)
```

Comment:
What's this do?

Reply:
This changes the behaviour when downloading via FetchContent to use the newest downloaded and extracted content when fetching for CMake 3.24+ (which visibly complains when it's not set)